test/ejabberd_tests/tests/ejabberdctl_SUITE.erl

```diff
@@ -720,3 +913,12 @@ string_to_binary(List) ->
         _ ->
             unicode:characters_to_binary(List)
     end.
+
+rpc_apply(M, F, Args) ->
```

Comment:
How about changing the name to `rpc_call`? Seams to be a better name, at least in my opinion.